### ISSUE TITLE
[BUG] ~ Links

### ISSUE THREAD

**Issue by dbarnes18** [ReplAPI-it/ReplAPI.it-NodeJS#65]
**Describe the bug**
A wrong link or not updated yet.

**To Reproduce**
1. Go to 'README.md'
2. Click on '[Explore the docs »](https://replapi-it.js.org/)'
4. This doesn't look like the right page

**Expected behavior**
Documentation on the ReplAPI.it

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] - macOS Catalina 10.15.7
 - Browser [e.g. chrome, safari] - Google Chrome | 91.0.4472.77&nbsp;(Official Build)&nbsp;(x86_64)

**Comment by dbarnes18** [comment]
Just noticed that this is for all the APIs not just the Node one. _Why am I reporting this on Node? I use Python 🤦_

**Comment by EpicGamer007** [comment]
Yeah, there has been a few problems with the docs, they will get back up as soon as possible.
